bedrock/newsletter/templatetags/helpers.py

```diff
@@ -70,6 +70,7 @@ def email_newsletter_form(
             success=success,
             email_label=email_label,
             email_placeholder=email_placeholder,
+            is_multi_newsletter_form=False if newsletters.find(",") == -1 else True,
```

Comment:
If you just need to know if there's a comma, not where, this is more commonly done in Python via `"," in newsletters`. Since that returns a boolean this line could be written as:

`is_multi_newsletter_form="," in newsletters`